src/kinsol/kinsol_ls.c

```diff
@@ -663,7 +659,7 @@ int kinLsATimes(void *kinmem, N_Vector v, N_Vector z)
   iterative linear solvers guarantee that kinLsPSetup will only
   be called in the case that the user's psetup routine is non-NULL.
   ---------------------------------------------------------------*/
-int kinLsPSetup(void *kinmem)
+SUNLsStatus kinLsPSetup(void *kinmem)
```

Comment:
This routine (and the next) conflates KINSOL-specific return values with `SUNLsStatus` values